src/cljc/athens/common_events/graph/atomic.cljc

```diff
@@ -79,12 +79,12 @@
 
 (defn make-page-rename-op
   "Creates `:page/rename` atomic op.
-   - `page-uid` - `:block/uid` of page to be renamed
+   - `old-name` - Page name before rename,
    - `new-name` - Page should have this name after operation"
-  [page-uid new-name]
+  [old-name new-name]
   {:op/type    :page/rename
    :op/atomic? true
-   :op/args    {:page-uid page-uid
+   :op/args    {:old-name old-name
```

Comment:
Interesting how with blocks we can just save by string + uid, but here we 100% need old and new name (because the content is the identity).